app/content/pages/index.html.md

```diff
@@ -26,7 +26,7 @@ Writing forces you to clarify your thinking of what's really important.
 
 ## Ship it!
 
-Once the landing page is complete its time to ship it! Ruby Monolith recommends the `dockerfile-rails` generatore and Fly.io.
+Once the landing page is complete its time to ship it! Ruby Monolith recommends the [`dockerfile-rails`](https://github.com/fly-apps/dockerfile-rails) generator and Fly.io.
```

Comment:
Thought I'd add a direct link to `dockerfile-rails` GitHub in case anyone wants to read more (like I did).